cartridge/feedback.lua

```diff
@@ -106,11 +156,8 @@ function _daemon_mt.__index.generate_feedback()
         return nil, err
     end
 
-    if feedback.rocks == nil then
-        feedback.rocks = {}
-    end
+    fill_in_rocks(feedback)
```

Comment:
What's the purpose of splitting this logic to a separate function?

Reply:
See [comment](https://github.com/tarantool/cartridge/pull/887#discussion_r441490797) above. Let's decide, which is more consistent for `cartridge` project.